### PR TITLE
address build warnings about ncurses example

### LINKED ISSUE #241: Address build warnings in ncurses example
Just a self-reminder really.
```
/home/runner/work/RF24Mesh/RF24Mesh/RF24Mesh/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp: In function ‘void drawTopology()’:
/home/runner/work/RF24Mesh/RF24Mesh/RF24Mesh/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp:219:56: warning: zero-length gnu_printf format string [-Wformat-zero-length]
  219 |     mvwprintw(topoPad, nodeY > 15 ? nodeY - 16 : 0, 0, "");
      |                                                        ^~
/home/runner/work/RF24Mesh/RF24Mesh/RF24Mesh/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp:241:25: warning: variable ‘y’ set but not used [-Wunused-but-set-variable]
  241 |                     int y = 0;
      |                         ^
/home/runner/work/RF24Mesh/RF24Mesh/RF24Mesh/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp:261:25: warning: variable ‘y’ set but not used [-Wunused-but-set-variable]
  261 |                     int y = 0;
      |                         ^
/home/runner/work/RF24Mesh/RF24Mesh/RF24Mesh/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp:281:25: warning: variable ‘y’ set but not used [-Wunused-but-set-variable]
  281 |                     int y = 0;
      |                         ^
```

### PR DESCRIPTION
**Pull request by 2bndy5**
resolves #241

### CODE REVIEW

**Review by TMRh20 — APPROVED** [review]
Very nice

File: examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp [hunk 1]
```diff
@@ -216,7 +216,7 @@ void drawTopology()
     wclear(topoPad);
     wattroff(topoPad, COLOR_PAIR(1));
     mvprintw(17, 10, "Mesh Topology");
-    mvwprintw(topoPad, nodeY > 15 ? nodeY - 16 : 0, 0, "");
+    mvwaddch(topoPad, nodeY > 15 ? nodeY - 16 : 0, 0, 0);
```
Comment (2bndy5) [inline examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp:219]: This change is my only concern. I want to be sure its behavior is as expected. Have you actually run this modified example?

Comment (TMRh20) [reply examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp:219]: Nope, I assumed you tested it hahaha

Comment (TMRh20) [reply examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp:219]: Just did a quick run and it looks the same. I think we're good

**Comment by 2bndy5** [comment]
According to the example's comment,
https://github.com/nRF24/RF24Mesh/blob/2fb4d25a4f789370b88063cfcf20a2cfe93cf6ff/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp#L14
If it is true, I think this needs to be updated to say
```cpp
  * NOTE: RF24MESH_DEBUG MUST BE DISABLED IN RF24Mesh_config.h 
```
I also noticed that every time I run this example without a pre-existing dhcplist.txt, a non-existent child node of ID 8 was assigned the address 01. Is this because I didn't enable `RF24MESH_DEBUG`?

**Comment by 2bndy5** [comment]
>I also noticed that every time I run this example without a pre-existing dhcplist.txt, a non-existent child node of ID 8 was assigned the address 01.

I found it.
https://github.com/nRF24/RF24Mesh/blob/2fb4d25a4f789370b88063cfcf20a2cfe93cf6ff/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp#L59

I suppose this is expected behavior given the above example-specific code.

**Comment by TMRh20** [comment]
https://github.com/nRF24/RF24Mesh/blob/2fb4d25a4f789370b88063cfcf20a2cfe93cf6ff/examples_RPi/ncurses/RF24Mesh_Ncurses_Master.cpp#L56
> 
> I suppose this is expected behavior given the above example-specific code.

Hmm, I thought we'd removed that line from all the examples. It should be removed, an old artifact from testing.

> If it is true, I think this needs to be updated to say
> ```c++
>   * NOTE: RF24MESH_DEBUG MUST BE DISABLED IN RF24Mesh_config.h 
> ```

Agreed. Can you add these changes to this PR?

**Comment by 2bndy5** [comment]
cherry picked to v1.x

**Pull request merged**